### PR TITLE
add plugin type to freshplug, freshdesk webwidget and zendesk app

### PR DESCRIPTION
**Pull request by dan753722**
plugin type is used when initializing iorad tutorial iframe for
analytics.

This is linked to issue #5

### CODE REVIEW

**Comment by dan753722** [comment]
doing a pull request to keep track of commits that are related to a certain issue.

File: IoradWebWidget/scripts/utils/utils.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ﻿ioradWebWidget.util.common = (function (module, win) {
+
+  /**
+   * this field is used to populate the options field when initializing iorad.js.
+   */
+  module.ioradPluginType = 'freshdeskWidget_tutorial_builder';
```
Comment (iorad) [inline IoradWebWidget/scripts/utils/utils.js]: freshdeskweb_solutions

File: freshplug/iorad_freshplug_integration.txt [hunk 2]
```diff
@@ -157,7 +157,7 @@
 
             listCategories().then(populateTutorialLocation, function (err) { });
 
-            iorad.init({ env: 'live' }, function () {
+            iorad.init({ env: 'live', pluginType: 'freshplug_tutorial_builder' }, function () {
```
Comment (iorad) [inline freshplug/iorad_freshplug_integration.txt]: freshplug_solutions

File: iorad_zendesk_app/lib/iorad.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 var env         = 'live', // cfdev, prod or live; default: live.
+  pluginType    = 'zendeskapp_tutorial_builder', // this is a nav bar app that creates tutorials.
```
Comment (iorad) [inline iorad_zendesk_app/lib/iorad.js]: zendeskapp_solutions

**Pull request merged**